src/analysis/special_functions/exp_log.lean

```diff
@@ -475,6 +475,17 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
+lemma log_inj_pos {x y : ℝ} (x_pos : 0 < x) (y_pos : 0 < y) (h : log x = log y) : x = y :=
+le_antisymm
+  ((log_le_log x_pos y_pos).1 $ h.le)
+  ((log_le_log y_pos x_pos).1 $ h.symm.le)
```

Comment:
I think this proof already exists as `strict_mono_incr_on_log.inj_on x_pos y_pos h`, if you move these three lemmas below line 498.

That raises that perhaps this lemma should actually be spelt and stated
```suggestion
lemma log_inj_on_pos : set.inj_on log (set.Ioi 0) :=
strict_mono_incr_on_log.inj_on
```
The proofs below should keep working after adjusting for the name change.